README.md

```diff
@@ -25,32 +25,37 @@ The latest source code is available at
 
 ## Project activity
 
-Story queue on Waffle:
-
-* [![Stories in Ready](https://badge.waffle.io/kartoza/ford3.svg?label=ready&title=Ready)](http://waffle.io/kartoza/ford3) 
-* [![Stories in In Progress](https://badge.waffle.io/kartoza/ford3.svg?label=in%20progress&title=In%20Progress)](http://waffle.io/kartoza/ford3)
-
-[![Throughput Graph](https://graphs.waffle.io/kartoza/ford3/throughput.svg)](https://waffle.io/kartoza/ford3/metrics)
-
-* Current test status master: [![Build Status](https://travis-ci.org/inasafe/inasafe.svg?branch=master)](https://travis-ci.org/inasafe/inasafe) and
+* Current test status master: [![Build Status](https://api.travis-ci.com/kartoza/ford3.svg?branch=master)](https://travis-ci.com/kartoza/ford3) and
 [![Code Health](https://landscape.io/github/kartoza/ford3/master/landscape.svg?style=flat)](https://landscape.io/github/kartoza/ford3/master)
 
-* Current test status develop: [![Build Status](https://travis-ci.org/inasafe/inasafe.svg?branch=develop)](https://travis-ci.org/inasafe/inasafe) and
+* Current test status develop: [![Build Status](https://api.travis-ci.com/kartoza/ford3.svg?branch=develop)](https://travis-ci.org/kartoza/ford3) and
 [![Code Health](https://landscape.io/github/kartoza/ford3/develop/landscape.svg?style=flat)](https://landscape.io/github/kartoza/ford3/develop)
 
 * Test coverage [![codecov](https://codecov.io/gh/kartoza/ford3/branch/develop/graph/badge.svg)](https://codecov.io/gh/kartoza/ford3)
 
-
-
 ## Quick Installation Guide
 
 For deployment we use [docker](http://docker.com) so you need to have docker 
 running on the host. Ford3 is a django app so it will help if you have
 some knowledge of running a django site.
 
+To run the project locally, there are four steps:
+1. Build the images and set up the docker env
+2. Populate initial data
+3. Run the server
+4. Open browser
+
+Suppose the current directory is `/home/web`.
+
+Therefore, the project will be in the `/home/web/ford3` directory.
+
+The project directory `/home/web/ford3` will be called and referred as `<parent>` throughout the rest of doc below.
+
+### 1. Build the images and set up the docker for the project
 ```
+cd /home/web
 git clone git://github.com/kartoza/ford3.git
-cd ford3/deployment
+cd *<parent>*/deployment
```

Comment:
why parent?
The name of the repo is `ford3` so `cd ford3/deployment` is correct

Reply:
it aims to shorten `ford3/deployment`.
i refer to this folder  in many lines.

do you want to use the long version instead?

Reply:
assume that the user is already in `ford3`